src/components/_videos.scss

```diff
@@ -0,0 +1,21 @@
+@use '../base.scss';
+
+.card--videos__iframe {
+  width: 100%;
+
+  margin-top: 0.8rem;
+}
```

Comment:
can you add `border: none;` to remove the frame from the iframes? Looks more modern :-)